src/table/cells/handle-key-press.js

```diff
@@ -1,5 +1,18 @@
 import { selectCell } from '../selections-utils';
 
+const isCtrlShift = (evt) => evt.shiftKey && (evt.ctrlKey || evt.metaKey);
+
+export const updatePage = (evt, totalRowSize, page, rowsPerPage, handleChangePage) => {
+  if (isCtrlShift(evt)) {
+    const lastPage = Math.floor(totalRowSize / rowsPerPage);
+    if (evt.key === 'ArrowRight' && page < lastPage) {
```

Comment:
If we set 'Rows per page' to 10 and `totalRowSize` is 40, it does not work as expected.

Reply:
true, fixed